### ISSUE TITLE
[NPC] Enraged Feral Scar

### ISSUE THREAD

**Issue by arcwizard** [SunwellWoW/Sunwell-TBC-Bugtracker#997]
Decription: 
[Enraged Feral Scar](https://tbc-twinhead.twinstar.cz/?npc=5295#abilities)

How it works: 
Frostbolt Volley has wrong dmg.
![wowscrnshot_102818_233039](https://user-images.githubusercontent.com/43764742/47684566-f9cbbd80-dbdb-11e8-81b8-1afd15df84ef.jpg)

How it should work: 
https://tbc-twinhead.twinstar.cz/?spell=8398

Source (you should point out proofs of your report, please give us some source): 
In discription.

P.S. other ability i can't check, cuz NPC kill me with frostbolt valey, and my countrespell all time resisted.

**Comment by 0vertime** [comment]
Yep, these guys can cast 3 volleys in row with their mana pool, and it hits for 700+.
